classes/Context.php

```diff
@@ -467,4 +467,17 @@ public function getComputingPrecision()
 
         return $this->priceComputingPrecision;
     }
+
+    /**
+     * Change context language and locale used in translator
+     *
+     * @since 1.7.8
+     *
+     * @param Language $language
+     */
+    public function setLanguageAndTranslatorLocale(Language $language)
```

Comment:
I'm not sure about this method : that breaks the principle of single responsability. Could you divide this method in two methods ?

Reply:
Fixed